draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -604,24 +649,26 @@
    A client sends requests to a server in the form of a <x:dfn>request</x:dfn>
    message with a method (<xref target="methods"/>) and request target
    (<xref target="target.resource"/>). The request might also contain
-   header fields (<xref target="header.fields"/>) for request modifiers,
-   client information, and representation metadata,
-   content (<xref target="content"/>) to be processed
+   header fields (<xref target="header.fields"/>) for request modifiers and
+   client information. If the method allows a representation to be included,
+   header fields might also include representation metadata. Any content
+   (<xref target="content"/>) present is processed
    in accordance with the method, and
-   trailer fields (<xref target="trailer.fields"/>) for metadata collected
+   trailer fields (<xref target="trailer.fields"/>) contain metadata collected
    while sending the content.
```

Comment:
Would it be easier to say that fields (i.e., header or trailer) could include representation metadata.

Reply:
Not really. This is supposed to be a really brief description. I change the trailers bit to be any information collected, since performance data and end-status aren't metadata.